### PR TITLE
docs: Added some documentation

### PR DESCRIPTION
**Pull request by criadoperez**
Added some documentation including:

- Link to new Token Economy working paper published yesterday
- Docker instructions to see logs and current status of bitcoin core sync
- Added a few necessary extra steps required to contribute

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,10 +48,12 @@ The theme is based on [docsify-themable](https://jhildenbiddle.github.io/docsify
 Contributions are what make the open source community such an amazing place to be learn, inspire, and create. Any contributions you make are greatly appreciated.
 
 1. Fork the Project
-2. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
-3. Commit your Changes (`git commit -m 'feat: Add some AmazingFeature'`)
-4. Push to the Branch (`git push origin feature/AmazingFeature`)
-5. Open a Pull Request
+2. Clone your new fork and enter the working folder (`git clone https://github.com/youruser/interlay.github.io.git && cd interlay.github.io`)
+3. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
+4. Make you changes
```
Comment (nud3l) [inline README.md]:
```suggestion
4. Make your changes
```

File: README.md [hunk 2]
```diff
@@ -48,10 +48,12 @@ The theme is based on [docsify-themable](https://jhildenbiddle.github.io/docsify
...
+5. Commit your Changes (`git commit -m 'feat: Add some AmazingFeature'`)
```
Comment (nud3l) [inline README.md]:
```suggestion
5. Commit your changes (`git commit -m 'feat: Add some AmazingFeature'`)
```

File: README.md [hunk 3]
```diff
@@ -48,10 +48,12 @@ The theme is based on [docsify-themable](https://jhildenbiddle.github.io/docsify
...
+6. Push to the Branch (`git push origin feature/AmazingFeature`)
```
Comment (nud3l) [inline README.md]:
```suggestion
6. Push to the branch (`git push origin feature/AmazingFeature`)
```

File: docs/vault/installation.md [hunk 4]
```diff
@@ -15,6 +15,7 @@ At the end of this document you will have:
 - Make sure you have at least 50 GB of free disk space (ideally SSD).
 - You should have a stable Internet connection.
 - Make sure that your Vault client is running for at least 8 hours per day (you can do other things on the side).
+- Make sure [docker](https://docs.docker.com/engine/install/ ) and [docker-compose](https://docs.docker.com/compose/install/) are installed in your system.
```
Comment (nud3l) [inline docs/vault/installation.md]: Docker and docker-compose are only required in the quickstart options that will not be available on Kintsugi. 

I suggest adding the following after line 26:

```
### 0. Install docker and docker-compose

Make sure [docker](https://docs.docker.com/engine/install/ ) and [docker-compose](https://docs.docker.com/compose/install/) are installed in your system.

File: docs/vault/installation.md [hunk 5]
```diff
@@ -56,17 +57,21 @@ If the Vault spends funds from another wallet this may be marked as theft.
...
 ?> You may need to edit the docker-compose to point `--bitcoin-rpc-url` to `http://localhost:18332`.
 
 You can run the entire Vault client and the Bitcoin node with the following command:
 
 ```shell
-docker-compose up
+docker-compose up -d
 ```
 
+You can optionally view the running docker containers with command `docker ps`.
```
Comment (nud3l) [inline docs/vault/installation.md]: Not 100% sure, but shouldn't docker-compose be sufficient? `docker-compose ps`, `docker-compose logs interbtc-vault_vault_1`, .. from the folder should work.

File: docs/kintsugi/overview.md [hunk 6]
```diff
@@ -2,6 +2,8 @@
 
 **Kintsugi** is interBTC's canary network, developed by Interlay under the supervision of Kintsugi Labs, a non-profit oriented organization responsible for Kintsugi's launch and the support of the decentralized network and ecosystem.
 
+The current Working Paper titled *Token Economy* is available [here](https://github.com/interlay/whitepapers/raw/master/Kintsugi_Token_Economy.pdf).
```
Comment (nud3l) [inline docs/kintsugi/overview.md]: This is already linked below see line 25. No need for this line.

**Review by nud3l — CHANGES_REQUESTED** [review]
Thanks for this contribution! I've left some comments

File: docs/vault/installation.md (hunk 4 above)
Comment (criadoperez) [reply docs/vault/installation.md]: Ok, better that way. I applied your suggestion.

File: docs/vault/installation.md (hunk 5 above)
Comment (criadoperez) [reply docs/vault/installation.md]: You can use both, docker and docker-compose for this task
I have however changed it to using docker-compose, if that's the one you like better.
The output of each one are slightly different and also the command changes:

With docker:
```
docker ps
docker logs interbtc-vault_vault_1 -f
docker logs interbtc-vault_bitcoind_1 -f
```

With docker-compose:
```
docker-compose ps
docker-compose logs -f vault
docker-compose logs -f bitcoind
```

File: docs/kintsugi/overview.md (hunk 6 above)
Comment (criadoperez) [reply docs/kintsugi/overview.md]: Ok, removed.

**Comment by criadoperez** [comment]
All changes completed. Should be ready for the merge.

**Review by nud3l — APPROVED** [review]

**Pull request merged**